static/js/components/CoverImageInput.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+import React from "react"
+import Dropzone from "react-dropzone"
+
+import { editorUpdateFormShim } from "./Editor"
+
+type Props = {
+  image: ?File | string,
+  onUpdate: Function,
+  setPhotoError?: Function
```

Comment:
Nit: `onPhotoError` might be a better name

Reply:
I think I'm going to stick with `setPhotoError` because in this component I'm setting the error string explicitly ("Please select a valid photo") - I think if it were an opaque function where this component just called it on the photo error then `onPhotoError` would make sense.